### PR TITLE
mtx: add commands

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

File: src/mtx/commands/downlink/setAccessKey.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+import * as types from '../../types.js';
+import CommandBinaryBuffer, {ICommandBinaryBuffer} from '../../utils/CommandBinaryBuffer.js';
+import * as command from '../../utils/command.js';
+import {READ_ONLY} from '../../constants/accessLevels.js';
+
+
+interface ISetAccessKeyParameters {
+    /**
+     * Access level from the list of [available levels](../../constants/accessLevels.ts).
```
Comment (DarkPark) [inline src/mtx/commands/downlink/setAccessKey.ts:36]: does this link work in docs?

File: src/mtx/commands/uplink/getExtendedCurrentValues.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import * as command from '../../utils/command.js';
+import CommandBinaryBuffer, {ICommandBinaryBuffer} from '../../utils/CommandBinaryBuffer.js';
+import * as types from '../../types.js';
+import {READ_ONLY} from '../../constants/accessLevels.js';
+
+
+interface IGetExtendedCurrentValuesResponseParameters {
+    /** device temperature */
+    temperature: types.TInt16,
+    /** The frequency of voltage in the power grid */
```
Comment (DarkPark) [inline src/mtx/commands/uplink/getExtendedCurrentValues.ts:36]:
```suggestion

    /** the frequency of voltage in the power grid */
```
or the prev description from upper

File: src/mtx/commands/downlink/setAccessKey.ts (hunk 1 above)
Comment (aleks-and) [reply src/mtx/commands/downlink/setAccessKey.ts]: reverted changes

File: src/mtx/commands/uplink/getExtendedCurrentValues.ts (hunk 2 above)
Comment (aleks-and) [reply src/mtx/commands/uplink/getExtendedCurrentValues.ts:37]: \+

**Review by DarkPark — APPROVED** [review]

**Pull request merged**